### ISSUE TITLE
Stable (non-alpha) releases

### ISSUE THREAD

**Issue by arminru** [open-telemetry/opentelemetry-proto-java#6]
The proto definitions for metrics and traces (including common and resources) are already defined as stable (see [opentelemetry-proto#maturity-level](https://github.com/open-telemetry/opentelemetry-proto#maturity-level)). The Java packages published from this repo here are still marked as `-alpha`, however, and as such also used in the stable opentelemetry-java SDK.
When will stable releases of opentelemetry-proto-java be available? What's still missing to consider it stable?
Thanks!

**Comment by anuraaga** [comment]
I don't think we want to go through the complexity of publishing two artifacts for stable and alpha, unless a user really asked for this. We would want all the protos to be marked Stable then, to release our artifact of all the signals to be stable. That means logs/* being marked Stable, and trace_config.proto being deleted (though we can workaround the latter here as we don't use it in the SDK, but someone really needs to get to deleting that :P)

**Comment by anuraaga** [comment]
> and as such also used in the stable opentelemetry-java SDK.

Ah by the way, we don't use the protos in the SDK itself, we only use them in tests. The SDK has its own marshaling logic that doesn't use the code published from here.
